lib/evm/git.rb

```diff
@@ -8,24 +8,36 @@ def exist?
       File.exist?(@path)
     end
 
-    def clone(url, branch = nil)
+    def clone(url, branch = nil, commit = nil)
       args = 'clone', url, @path
       args << "--branch=#{branch}" if branch
+     if commit == nil
```

Comment:
I think `nil?` is more Ruby-ish? Maybe cleaner to pass this in as a boolean instead of the (unused) actual commit?